ClLike/cl_like/ccl.py

```diff
@@ -114,13 +115,15 @@ def calculate(self, state, want_derived=True, **params_values_dict):
                               T_CMB=2.7255,
                               transfer_function=self.transfer_function,
                               matter_power_spectrum=self.matter_pk,
-                              baryons_power_spectrum=self.baryons_pk)
+                              baryons_power_spectrum=self.baryons_pk,
+                              **self.ccl_arguments)
 
         state['CCL'] = {'cosmo': cosmo}
 
         # Compute derived parameters
         # (we should actually only do this if required -- TODO)
         # Compute sigma8 if it is not an input parameter
+        state['derived'] = {}
```

Comment:
Shouldn't line 129 be `state['derived']['sigma8'] = sigma8` then?

Reply:
Yeah. It will not matter at the end of the day because nothing is happening in between but yes. I added L126 the other day after failing running some chains